ui/pages/settings/security-tab/security-tab.component.js

```diff
@@ -83,24 +85,26 @@ export default class SecurityTab extends PureComponent {
     );
   }
 
-  renderMetaMetricsOptIn() {
+  renderIncomingTransactionsOptIn() {
```

Comment:
github diff checker is a bit confused on this file, I simply reordered the render functions on this file and added a new one